packages/blade/src/components/Input/BaseInput/StyledBaseInput.web.tsx

```diff
@@ -70,13 +70,15 @@ export const StyledBaseInput = React.forwardRef<HTMLInputElement, StyledBaseInpu
       autoCompleteSuggestionType,
       accessibilityProps,
       setCurrentInteraction,
+      type,
       ...props
     },
     ref,
   ) => {
     return (
       <StyledBaseNativeInput
         ref={ref}
+        type={type === 'telephone' ? 'tel' : type}
```

Comment:
I think this would need a patch release?

Reply:
yes. will add changesets.